src/test/java/com/epam/tc/hw2/BaseData.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.tc.hw2;
+
+public class BaseData {
+
+    public static final String PAGE_URL = "https://jdi-testing.github.io/jdi-light/index.html";
+    public static final String HOME_PAGE_TITLE = "Home Page";
+    public static String LOGIN = "Roman";
+    public static String PASSWORD = "Jdi1234";
+    public static String USERNAME = "ROMAN IOVLEV";
```

Comment:
All these data can be moved in property file

Reply:
I moved them into a property file, created PropertyReader class and deleted BaseData class